src/coreclr/System.Private.CoreLib/src/System/Threading/Overlapped.cs

```diff
@@ -120,9 +85,6 @@ internal sealed unsafe class OverlappedData
 
         [MethodImpl(MethodImplOptions.InternalCall)]
         internal static extern OverlappedData GetOverlappedFromNative(NativeOverlapped* nativeOverlappedPtr);
```

Comment:
How about porting the rest of the `Overlapped` fcalls to managed code? Especially `GetOverlappedFromNative` whose implementation is trivial (just casting a pointer and a couple of dereferences) would be benefitted from being inlined by the JIT.

Reply:
Yes, there are more follow up improvements that can be done here. Not as part of this PR through.